src/graph/executor/query/RollUpApplyExecutor.cpp

```diff
@@ -121,7 +121,10 @@ DataSet RollUpApplyExecutor::probe(std::vector<Expression*> probeKeys,
     List list;
     list.values.reserve(probeKeys.size());
     for (auto& col : probeKeys) {
-      Value val = col->eval(ctx(probeIter));
+      // Clone the expression so when evaluating the InputPropertyExpression, the propIndex_ will
```

Comment:
I think you don't need to clone for each row, just clone once for `build` and `probe`.

Reply:
You are right, I'll fix the copy part.